spec/services/payments/create_funds_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+module Payments
+  describe CreateFunds do
+    describe '.call' do
+      context 'with positive amount' do
+        it 'add funds to a cashbox' do
+          cashbox = Cashbox.create
+
+          res = CreateFunds.call(cashbox, 1000)
+
```

Comment:
should we also be checking `expect(res).to be_truthy`, or is that not an intended behaviour?